catalogs/catalog-lakehouse-iceberg/src/main/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/IcebergConfig.java

```diff
@@ -76,6 +78,29 @@ public class IcebergConfig extends Config {
           .booleanConf()
           .createWithDefault(true);
 
+  public static final ConfigEntry<String> ICEBERG_METRICS_STORE =
+      new ConfigBuilder(IcebergMetricsManager.ICEBERG_METRICS_STORE)
+          .doc("The store to save Iceberg metrics")
+          .version("0.4.0")
+          .stringConf()
+          .createWithDefault(BlackHoleMetricsStore.ICEBERG_METRICS_STORE_BLACK_HOLE_NAME);
```

Comment:
I would suggest to make this configuration as no default value, which means not storing any metrics, If user configures this, then we will use the configured one.

Reply:
if user doesn't configurate it, the value will be null, we convert null to dummyMetricsStore which doesn't store any metrics right?

Reply:
> if user doesn't configurate it, the value will be null, we convert null to dummyMetricsStore which doesn't store any metrics right?

yes.

Reply:
Please change it.